build.gradle

```diff
@@ -46,17 +46,24 @@ if (!Boolean.getBoolean("skipSonarqube")) {
     apply plugin: "org.sonarqube"
 }
 
+configurations {
+    codacy
+}
+
 buildscript {
     repositories {
         mavenLocal()
         jcenter()
+        maven { url "https://jitpack.io" }
+        maven { url "http://dl.bintray.com/typesafe/maven-releases" }
         maven { url "https://maven.eveoh.nl/content/repositories/releases" }
         maven { url "https://plugins.gradle.org/m2/" }
         maven { url "https://repo.spring.io/plugins-release" }
         maven { url "https://repo.spring.io/libs-milestone" }
     }
 
     dependencies {
+        classpath 'com.github.codacy:codacy-coverage-reporter:-SNAPSHOT'
```

Comment:
Could we instead build against a specific commit id? or entirely remove jitpack and build against a published binary? (The service generally is very flaky).

Reply:
PS Also please keep consistent with other classpath entries by using `"` instead of `'`